src/views/GovernanceReviewTeam/Actions/RejectIntake.tsx

```diff
@@ -13,16 +18,21 @@ import HelpText from 'components/shared/HelpText';
 import Label from 'components/shared/Label';
 import TextAreaField from 'components/shared/TextAreaField';
 import { RejectIntakeForm } from 'types/action';
-import { rejectSystemIntake } from 'types/routines';
 import flattenErrors from 'utils/flattenErrors';
 import { rejectIntakeSchema } from 'validations/actionSchema';
 
 const RejectIntake = () => {
   const { systemId } = useParams<{ systemId: string }>();
-  const dispatch = useDispatch();
   const history = useHistory();
   const { t } = useTranslation('action');
 
+  const [mutate, mutationResult] = useMutation<
+    RejectIntakeType,
+    RejectIntakeVariables
+  >(RejectIntakeQuery, {
+    errorPolicy: 'all'
```

Comment:
Let's talk more about this next week to see if it's something we want to do globally for all mutations.